llvm/test/CodeGen/PowerPC/aix-cc-abi-mir.ll

```diff
@@ -1,8 +1,8 @@
 ; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
-; RUN: llc -mtriple powerpc-ibm-aix-xcoff -stop-after=machine-cp -verify-machineinstrs < %s | \
+; RUN: llc -mtriple powerpc-ibm-aix-xcoff -mcpu=generic -stop-after=machine-cp -verify-machineinstrs < %s | \
```

Comment:
It does not make sense to me to set `-mcpu=generic` with this patch for all LIT cases. Unless the test point is broken with the CPU change, otherwise we should update the cases.